### DISCUSSION TITLE
Sigrok support for FSCOPE-500K

### DISCUSSION THREAD

**Discussion by Fabian-Schmidt** [fhdm-dev/scoppy#138] [General]
Hi,
Sigrok / Pulseview supports since 2023-09-27 a bare Raspberry Pi Pico as a logic analyzer and oscilloscope. 

Is it possible to use FSCOPE-500K as a analog frontend for this?

So in the field when I want to quickly check I use Android with Scoppy and for detailed analysis with protocols I can use sigrok.

I have not found any schematics for FSCOPE-500K to identify which signals the RP2040 provides to make it work.

Links:
- Original repo of project and firmware sigrok-pico needed for sigrok support <https://github.com/pico-coder/sigrok-pico>
- PR merged <https://github.com/sigrokproject/libsigrok/pull/181>
- GIT commit of support <https://sigrok.org/gitaction/libsigrok.git/8e3f798..3c2db8a>

Thanks

**Comment by fhdm-dev** [comment]
Hi,

Sorry but I don't know if the FSCOPE-500K could be used with the sigrok-pico firmware.
The schematic can be found here:

[OSHWLab - FSCOPE-500K Rev 4e](https://oshwlab.com/fruitloop57/fscope-250k5-v2_copy_copy_copy_copy)

:)
